app/models/condition_set/conditions_extension.rb

```diff
@@ -7,19 +7,13 @@ def sorted
     end
 
     def standard
-      standard_conditions.map do |condition|
+      Condition.standard_conditions.map do |condition|
```

Comment:
Trying to work out why we don't do `select(&:standard?)` here - legacy data issue maybe?

Reply:
I'm actually not sure if that's used at all now.